src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -744,14 +744,14 @@ def test_vm_create_by_attach_unmanaged_os_and_data_disks(self, resource_group):
     def test_vm_create_data_disk_delete_option(self, resource_group):
         self.cmd('vm create -n Delete_CLI1 -g {rg} --image RedHat:RHEL:7-RAW:7.4.2018010506 -l northeurope '
                  '--size Standard_E8as_v4 --generate-ssh-keys --public-ip-address "" --os-disk-size-gb 64 '
-                 '--data-disk-sizes-gb 200 --data-disk-delete-option Delete',
+                 '--data-disk-sizes-gb 200 --data-disk-delete-option Delete --admin-username vmtest',
```

Comment:
>  --admin-username vmtest

Why do we need to add this parameter to those tests?

Reply:
If `--admin-user` is not specified, test will fail with `admin user name cannot contain upper case character A-Z, special characters \/"[]:|<>+=;,?*@#()! or start with $ or -`

Reply:
Why didn't this problem happen before?

Reply:
If `admin-username` is not specified, the local machine name will be used as input by default. Specifying `admin-username` ensures that it meets the conditions (`admin user name cannot contain upper case character A-Z, special characters \/"[]:|<>+=;,?*@#()! or start with $ or -`).